src/hyperfine/benchmark.rs

```diff
@@ -358,16 +371,18 @@ pub fn run_benchmark(
 
     let (user_str, user_unit) = format_duration_unit(user_mean, None);
     let system_str = format_duration(system_mean, Some(user_unit));
+    let max_rss_str = format!("{:.3} MB", max_rss_mean);
```

Comment:
I would prefer if we would show either KiB or MiB (depending on the actual usage), with fewer digits in the fractional part.

Reply:
Yes, probably a `format_memory` function would make that better :)